layers/install/stacker.yaml

```diff
@@ -10,17 +10,33 @@ install-rootfs-pkg:
     type: built
     tag: install-base
   run: |
+    #!/bin/sh -ex
     pkgtool install \
         cryptsetup \
         dosfstools \
         e2fsprogs \
+        iproute2 \
+        isc-dhcp-client \
         keyutils \
         kmod \
         libsquashfs-dev \
         parted \
         tpm2-tools \
         udev
 
+    writefile() {
+      mkdir -p "${1%/*}"
+      echo "write $1" 1>&2
+      cat >"$1"
+    }
+
+    writefile /etc/systemd/network/20-wire-enp0s-dhcp.network <<"END"
```

Comment:
only dhcp?

Reply:
This was inherited from a bootkit script.  It's not what we want.  But as it's just the demo rootfs, and we want to update that anyway to include an assert service nad probably an external-facing zot service, I figure we can fine-tune that later.